src/core/ColorPicker/ColorSwatch.tsx

```diff
@@ -22,15 +22,16 @@ export type ColorSwatchProps = {
 /**
  * ColorSwatch component to display a color within a basic color picker.
  * @example
- * <ColorSwatch color='#23450b'  onClick={onClick}/>
+ * <ColorSwatch color={ColorValue.create('#23450b'}  onClick={onClick}/>
  */
 export const ColorSwatch = React.forwardRef<HTMLDivElement, ColorSwatchProps>(
   (props, ref) => {
     const { color, style, onClick, isActive, className, ...rest } = props;
+    const inColor = getColorValue(color);
```

Comment:
Using this function here might not be correct behavior actually. This function (and `ColorValue.create`) don't support all valid css color formats. Examples include keywords like `transparent` and `currentColor`, as well as the new [space-separated functional notation from CSS Colors Level 4 spec](https://www.w3.org/TR/css-color-4/#:~:text=new%20syntax%20consisting%20of%20space-separated%20arguments%20and%20an%20optional%20slash-separated%20opacity).

So it might be better to just assign the value to the style if it's not an instance of `ColorValue`.